doc/api/os.md

```diff
@@ -215,6 +215,15 @@ added: v2.3.0
 The `os.homedir()` method returns the home directory of the current user as a
 string.
 
+POSIX:
+Will use the `$HOME` environment variable if defined. Otherwise, it will use
+the [effective UID](https://en.wikipedia.org/wiki/User_identifier#Effective_user_ID)
```

Comment:
@bnoordhuis Wikipedia seems to be a better option, Effective user id is better explained there; I choosed that one over the [stackoverflow reference](https://stackoverflow.com/a/32456814) because making a link of "effective UID", by reading, looks like a definition instead of "comparison" or "differences" between the others UID models in POSIX.

By the way, I found a [paper](https://people.eecs.berkeley.edu/~daw/papers/setuid-usenix02.pdf), take a look at the page 2, "User ID Model"; also the [POSIX docs](http://pubs.opengroup.org/onlinepubs/9699919799/), on Definitions section, the definition number 3.142 corresponds to "Effective user ID"

What do you think?

Reply:
Although I personally prefer inline links like you've done here (because it keeps the raw markdown more usable IMO), the docs tend to use referenced links (which are at the bottom of the doc in alphabetical order). I'm fine with this landing as-is, but if you want to make it a reference link for consistency, that's cool too.

Reply:
I'm on it!